test_unstructured_ingest/expected-structured-output/delta-table/0-9d594ee0-ad36-4e7e-a6be-f53975fe3d10-8.json

```diff
@@ -5,8 +5,8 @@
     "metadata": {
       "data_source": {
         "url": "s3://utic-dev-tech-fixtures/sample-delta-lake-data/deltatable/0-9d594ee0-ad36-4e7e-a6be-f53975fe3d10-8.parquet",
-        "date_created": "2023-08-16 13:30:38.644000",
-        "date_modified": "2023-08-16 13:30:38.644000"
+        "version": 329407810704817028643559273505069222621,
+        "date_modified": "2023-08-16T13:32:11"
```

Comment:
looks like we lost date_created?

Reply:
> we define date_modified for all ingest docs generated by the connector at line 137
in what file? interfaces? what is the pro/con of the change

ALSO: that's unrelated to my actually question, which is why we no longer have `date_created`